documentation/book/con-kafka-mirror-maker-authentication.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-authentication.adoc
+
+[id='con-kafka-mirror-maker-authentication{context}']
+= Authentication support in Kafka Mirror Maker
+
+Authentication can be configured in the `KafkaMirrorMaker.spec.consumer.authentication` and `KafkaMirrorMaker.spec.producer.authentication` properties.
+The `authentication` property specifies the type of the authentication method which should be used and additional configuration details depending on the mechanism.
+The currently supported authentication types are:
+
+* TLS client authentication
+* SASL based authentication using SCRAM-SHA-512 mechanism
+
+
+== TLS Client Authentication
+
+To use the TLS client authentication, set the `type` property to the value `tls`.
```

Comment:
Rewrite to:

To use the TLS client authentication, set the type property to the value `tls`. The TLS client authentication uses TLS certificate to authenticate. The certificate has to be specified in the certificateAndKey property. It is always loaded from a {ProductPlatformName} secret. Inside the secret, it has to be stored in the X.509 format separately as public and private keys.